src/component/menu/Header/NavigationLink/NavigationLink.styles.ts

```diff
@@ -23,6 +23,6 @@ export const useStyles = makeStyles(theme => ({
         color: '#000',
         height: '100%',
         width: '100%',
-        padding: '0.5rem 1rem',
+        padding: '0.5rem 1rem !important',
```

Comment:
Ooh, `!important`? Can we increase the specificity of this selector some other way to avoid this?

Reply:
Good idea. We can use the `classes` prop to override MUI styles instead.

Reply:
Good idea, we can use the `classes` prop to override MUI styles instead.